springboot/src/main/resources/db/migration/V1.2__Description.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE permission (
+	id INTEGER NOT NULL PRIMARY KEY,
+	name VARCHAR(50) NOT NULL,
+	description VARCHAR(250) NULL,
+	deleted BOOLEAN NOT NULL DEFAULT FALSE
+);
+
+CREATE TABLE role (
+	id INTEGER NOT NULL PRIMARY KEY,
+	name VARCHAR(50) NOT NULL,
+	description VARCHAR(250) NULL,
+	deleted BOOLEAN NOT NULL DEFAULT FALSE
+);
+
+CREATE TABLE role_permission (
+	id_role INTEGER NOT NULL,
+	id_permission INTEGER NOT NULL,
+	PRIMARY KEY (id_role, id_permission),
+	CONSTRAINT role_permission_role_FK FOREIGN KEY (id_role) REFERENCES role (id),
+	CONSTRAINT role_permission_permission_FK FOREIGN KEY (id_permission) REFERENCES permission (id)
+);
+
+CREATE TABLE public.user (
+	id INTEGER NOT NULL PRIMARY KEY,
+	name VARCHAR(50) NOT NULL,
```

Comment:
O limite do campo nome, na US é 250, se colocar o VARCHAR(50) ele aceita um número maior que 50?
https://github.com/eduardopessoa89/Point-of-Sale/issues/5

Reply:
Não. Mas foi corrigido pra 255 caracteres.